hbase-client/src/main/java/org/apache/hadoop/hbase/client/Table.java

```diff
@@ -751,4 +751,12 @@ default long getWriteRpcTimeout(TimeUnit unit) {
   default long getOperationTimeout(TimeUnit unit) {
     throw new NotImplementedException("Add an implementation!");
   }
+
+  /**
+   * Get the attributes to be submitted with requests
+   * @return map of request attributes
+   */
+  default Map<String, byte[]> getRequestAttributes() {
+    return Collections.emptyMap();
```

Comment:
Might be better to throw an exception. Neither option is great if someone actually ends up calling the default, but I think throwing a clear exception is better than potentially returning the wrong results (i.e. there may indeed by request attributes).